core/src/main/scala/quasar/destination/avalanche/package.scala

```diff
@@ -115,27 +115,31 @@ package object avalanche {
         .queryWithLogHandler[Int](logHandler)
     }
 
+    def grantAllTableUpdate: Update0 =
+      (fr"GRANT ALL PRIVILEGES ON TABLE" ++ Fragment.const(tableName) ++ fr"TO group dbadmingrp")
```

Comment:
Should `group dbadmingrp` be configurable, with a default to this value?

Reply:
@alissapajer 

This was discussed with @beckyconning to keep minimal and hardcode for now. She also mentioned that we could make it optional, but we decided for the minimal approach instead. 

If there are requirements to have things more configurable in the future, there are various other options doing that (apart from your and Becky's suggestion), e.g.
- allow the user to define the complete statement 
- create a templating system with DSL so that statements like this are completely configurable (e.g. like we are doing right now in electron for H2).

But yeah chances for YAGNI, so that's why the minimum approach here.

Reply:
"Hardcoding" in the sense that it isn't user-configurable is fine, but we could still parameterize on the group name in the implementation so that it is specified near the rest of the configuration instead of in this library function.

Reply:
Agreed that it would be preferable to parameterize on the group name so it's clear that this is a "parameter" and not a true "constant".

Reply:
Hmm, I don't know. The `ALL PRIVILEGES` and the complete `GRANT` statement can also be considered as parameter, but how/if we will actually parameterize depends on future requirements.

What do you think about extracting constants for `ALL PRIVILEGES` and `group dbadmingrp`, but without introducing extra parameters?

Reply:
What we consider as parameters depends on how much we're trying to generalize, my suggestion was intended to preserve the minimal behavior of "all privileges will be granted on the new table to `<group>`" and just push where `<group>` is defined out further, in an effort to keep things explicit at much as possible.

Anyhow, pulling the group name into a constant would be an improvement in discoverability, if nothing else 👍